Makefile

```diff
@@ -0,0 +1,28 @@
+
+help:
+	@echo "Libero reviewer CI Makefile"
+	@echo "---"
+	@echo "The purpose of this file is to help clean up the ci-config"
+	@echo "and to make it easier to switch CI providers if necessary"
+	@echo ""
+	@echo "To run the CI script locally, use `make all`, which will
+	@echo "start building all components in parallel"
+
+build-server:
```

Comment:
Can we rename these - its not just a build ... its a build and test :smile:

Reply:
sure